compiler-rt/lib/sanitizer_common/sanitizer_stacktrace_printer.cpp

```diff
@@ -59,8 +45,27 @@ const char *FormattedStackTracePrinter::StripFunctionName(
   return function;
 }
 
-// sanitizer_symbolizer_markup.cpp implements these differently.
-#if !SANITIZER_SYMBOLIZER_MARKUP
+// sanitizer_symbolizer_markup_fuchsia.cpp implements these differently.
+#if !SANITIZER_SYMBOLIZER_MARKUP_FUCHSIA
+
+StackTracePrinter *StackTracePrinter::GetOrInit() {
```

Comment:
Does wrapping this with `SANITIZER_SYMBOLIZER_MARKUP_FUCHSIA` mean it won't be defined if we're not using symbolizer markup?

Reply:
It means it won't be used if we are compiling for fuchsia, this one was part of the motivation on renaming this definition. `StackTracePrinter::GetOrInit` is implemented a bit different when compiling for fuchsia `compiler-rt/lib/sanitizer_common/sanitizer_symbolizer_markup_fuchsia.cpp:26`